### ISSUE TITLE
Warzone Teleports Bug

### ISSUE THREAD

**Issue by rikitrini** [orts/server#128]
Hello guys, well i checked alot of things about Gnomission, storage values and all but after i say to him mission and they say to me Fine i grant acces to warzones, i still can't enter to the warzones someone got any fix for that? if someone can help me^^ thanks for all

**Comment by EPuncker** [comment]
do you have enough gnome reputation?

**Comment by rikitrini** [comment]
@EPuncker yes for sure mate i have 2000 of reputation, i make done all tasks too and i reported to commander stone and reported to gnomeral and they say congragulations you up rank ect.. ect.. and i go to gnomission i say to him warzones, mission ect.. they say fine i grant acces but still can't enter warzones  i paste to you my scripts bro maybe u can help me 
WarzoneTeleports.lua:http://pastebin.com/BmViej9Y
Gnomission:http://pastebin.com/Ry2SQsM3
051-storages:http://pastebin.com/FUJmAgEN
thanks for all
